.github/workflows/integration-tests.yml

```diff
@@ -105,3 +109,16 @@ jobs:
             -k "not(builtin_tool or safety_with_image or code_interpreter or test_rag)" \
             --text-model="meta-llama/Llama-3.2-3B-Instruct" \
             --embedding-model=all-MiniLM-L6-v2
+
+      - name: Write ollama logs to file
+        run: |
+          sudo journalctl -u ollama.service > ollama.log
+
+      - name: Upload all logs to artifacts
+        if: always()
```

Comment:
for my education, what does this do? is this not equivalent to not having this conditional at all?

Reply:
This will collect logs on both success and failure :)

Reply:
@leseb TIL!